cpp/include/Ice/SSL/Exception.h

```diff
@@ -0,0 +1,73 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+#ifndef ICE_SSL_EXCEPTION_H
+#define ICE_SSL_EXCEPTION_H
+
+#include "../Config.h"
+#include "../Exception.h"
+
+#include <string>
+
+namespace Ice::SSL
+{
+    /**
+     * Thrown if the certificate cannot be read.
+     */
+    class ICE_API CertificateReadException : public Ice::Exception
+    {
+    public:
+        using Ice::Exception::Exception;
+
+        CertificateReadException(const char*, int, std::string) noexcept;
+
+        std::string ice_id() const override;
+
+        /** The reason for the exception. */
+        std::string reason;
+
+    private:
+        static const char* _name;
```

Comment:
This _name doesn't appear to be used anywhere. Or defined.

Reply:
Removed